.github/ISSUE_TEMPLATE/public_access_content_review.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Review publicly accessible content
+about: 'Issue template for the system administrator to perform a review of publicly accessible content.'
```

Comment:
Why is this quoted?

Reply:
Convention. Since all the other issue templates use quotes for this property, I figured that was the standard.

Reply:
I can't think of a reason as to why we would have that convention. I think this is just incidental. 

Please remove all unnecessary quotes from these templates in a separate commit.